providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/FlagdProvider.java

```diff
@@ -184,19 +184,6 @@ public ProviderEvaluation<Value> getObjectEvaluation(String key, Value defaultVa
         return flagResolver.objectEvaluation(key, defaultValue, ctx);
     }
 
-    /**
-     * An unmodifiable view of a Structure representing the latest result of the
-     * SyncMetadata.
-     * Set on initial connection and updated with every reconnection.
-     * see:
-     * https://buf.build/open-feature/flagd/docs/main:flagd.sync.v1#flagd.sync.v1.FlagSyncService.GetMetadata
-     *
-     * @return Object map representing sync metadata
-     */
-    protected Structure getSyncMetadata() {
-        return new ImmutableStructure(eventsLock.syncMetadata.asMap());
-    }
-
```

Comment:
This is the "raw" metadata which was a protected prop before. TBH I don't think we need it. What do you guys think?